SITE_CONTRIBUTION.md

```diff
@@ -0,0 +1,134 @@
+├── _resources # General information pages
+└── _scss # Site style sheets
+    ├── bootstrap-4.1.3
+    │   ├── mixins
+    │   └── utilities
+    └── site
+        ├── common
+        ├── layouts
+        ├── objects
+        ├── settings
+        └── utilities
+```
+
+## Link Insertion Standards
+
+Usage of links within a page has been standardized to simplify the process of finding and fixing links as required.
+
+Avoid using inline links, instead, use a reference table the bottom of the file; making it easier to find and update links. [Reference Links][3]. There are some exceptions to this rule, such as depreciation warnings.
+
+There are several conditions which will determine to correct type of link templating to use.
+
+**Linking to a blog post:**
+
+Use this formation when linking to a projectcontour.io blog post. `{% post_url 2010-07-21-name-of-post %}`
+
+This method will follow any site / page changes to permalinks and Jekyll will validate the link during build.
+
+**Linking to a projectcontour.io page:**
+
+This template is used when linking to another projectcontour.io page. Path to the .md is relative from /site/.
+`{% link getting-started.md %}` `{% link docs/v1.0.0/deploy-options.md %}`
+
+Jekyll will perform validation against links inserted with this method.
+
+**Linking to a section within the same page:**
+
+Links are created for all headers when the site is built. You can link to sections within the same page by using `#section-text`.
+Spaces within a heading are replaced with a hyphen. `## My heading` has a link `#My-heading`.
+
+**Linking to a section within another page:**
+
+Unfortunately, the `{% link %}` templating method does not allow linking to a section within a page. Use this format when linking to a section within another page.
+`{% <Path to page post generation/<section> %}` or `/docs/v1.0.0/deploy-options/#Host-Networking`
+
+A key difference between using `{% link %}` and the above method is that `{% link %}` will resolve to pages final location. To use the above method, you need to point the link to the location where the document will end up. For example, files in the `_guides` directory will be located in `/guides` on the live site. Therefore, the link for `_guides/cert-manager` is `/guides/cert-manager`.
+
+Additionally, the above method will not follow any permalink changes or page-specific permalink.
+
+**Linking to the project contour GitHub repo:**
+
+The GitHub metadata plugin populates Jekyll variables with data from GitHub during build. When inserting a link to the Project Contour repository, use the prefix `{{site.github.repository_url}}`.
+
+A full list of variables can be found in the [jekyll-github-metadata documentation][4].
+
+**Linking to an external page:**
+
+Links to external pages are done using the full URL, `[1]: https://projectcontour.io` for example.
+
+## Proofread and spell check
+
+If you're making any content submissions, please proofread and spell check your work.
```

Comment:
Maybe note here that running `make check` will spell check for you?

Reply:
I'll add a note, though I would like people to do their own checking as well